tests/TestCase/Model/Behavior/ProfferBehaviorTest.php

```diff
@@ -157,6 +157,10 @@ public function beforeMarshalProvider()
 
     /**
      * @dataProvider beforeMarshalProvider
+     *
+     * @param array $data
```

Comment:
I've not seen that notation anywhere, but it's a good idea. Is that supported by docblock standards?